u_fingerprinter.go

```diff
@@ -0,0 +1,302 @@
+// Copyright 2017 Google Inc. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
```

Comment:
I assume this is the proper header?

Reply:
Meh, good enough.
Probably the right way is to create UTLS_CONTRIBUTORS file, put our past and present contributors there, and refer to that file, as I don't want to overload code headers with the names of Tom, Dick and Harry, but this is simpler and will do.
Doesn't really matter